src/pages/borrow.tsx

```diff
@@ -51,6 +54,27 @@ const Borrow: NextPage = () => {
           <ReserveDropdown action="borrow" poolId={safePoolId} activeReserveId={safeAssetId} />
         </Section>
       </Row>
+      <Row>
+        <Section width={SectionSize.FULL}>
+          <LinkBox
+            sx={{ width: '100%' }}
+            to={{ pathname: '/asset', query: { poolId: poolId, assetId: safeAssetId } }}
+          >
+            <CustomButton
+              sx={{
+                width: '100%',
+                alignItems: 'space-between',
+                paddingLeft: '12px',
+                paddingRight: '12px',
+                '&:hover': { backgroundColor: theme.palette.background.default },
+              }}
+            >
+              <Typography variant="body1">Asset Info</Typography>
+              <ArrowForwardIcon fontSize="inherit" sx={{ color: theme.palette.text.secondary }} />
+            </CustomButton>
+          </LinkBox>
+        </Section>
+      </Row>
```

Comment:
I think this looks pretty weird.

Can we consider doing something like the pool header dropdown (and like the comment on the asset page), except this one would link to the assets page?

```
[ Reserve Dropdown ] [ Details ]
```

Reply:
For mobile, we could consider an arrow instead of an external link `[ -> ]` or maybe the info icon.